bot/exts/filters/token_remover.py

```diff
@@ -78,6 +83,9 @@ async def on_message(self, msg: Message) -> None:
         if found_token:
             await self.take_action(msg, found_token)
```

Comment:
We could add a return after this - there's no point in checking for an MFA token.

Reply:
I think it is worth checking, or I should move checking for an mfa token before checking the other token format.

the reason for checking both was to ensure that a user who happened to share a user and bot token in the same message would not go undetected.

Reply:
Basically, with an mfa token those are to alert mods that someone sent it,  and don't let the user know why their messages was deleted intentionally.

however, I should probably go a little further with the implementation and possibly add some configuration as we may not want to ping moderators for mfa tokens

Reply:
*Reading back my mails, I realize that it might be best for me to respond to this..*

Makes sense yeah, we'll keep it like this then.